tests/unit/src/test/scala/tests/troubleshoot/ProblemResolverSuite.scala

```diff
@@ -60,30 +58,16 @@ class ProblemResolverSuite extends FunSuite {
 
   checkRecommendation(
     "unsupported-sbt-version",
-    scalaVersion = "2.12.7",
-    UnsupportedSbtVersion.message,
+    scalaVersion = BuildInfo.scala212,
+    UnsupportedSbtVersion("1.2.0").message,
     sbtVersion = Some("1.2.0")
   )
 
-  checkRecommendation(
-    "deprecated-sbt-version",
-    scalaVersion = "2.12.8",
-    DeprecatedSbtVersion.message,
-    sbtVersion = Some("1.3.0")
-  )
-
-  checkRecommendation(
-    "future-sbt-version",
-    scalaVersion = "2.12.51",
-    FutureSbtVersion.message,
-    sbtVersion = Some("1.6.0")
-  )
```

Comment:
can we leave this test to check `2.12.51` is reported as `UnsupportedScalaVersion`?